spock-container-test-app/build.gradle

```diff
@@ -1,67 +1,66 @@
-plugins {
-        id "groovy"
-        id "org.grails.grails-web"
-        id "org.grails.grails-gsp"
-        id "war"
-        id "idea"
-        id "com.bertramlabs.asset-pipeline"
-        id "eclipse"
+buildscript {
+    repositories {
+        maven { url = 'https://repo.grails.org/grails/core/' }
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:$assetPipelineVersion"
+        classpath "org.grails:grails-gradle-plugin:$grailsVersion"
+    }
 }
 
-group = "org.demo.spock"
+apply plugin: 'groovy'
+apply plugin: 'org.grails.grails-web'
+apply plugin: 'org.grails.grails-gsp'
```

Comment:
Should we be only applying the grails plugin gradle plugin?

Reply:
This build file is for the functional tests project, which is a grails app.